Automattic/wp-calypso
# Add monthly pricing to My Plans page

#### Changes proposed in this Pull Request

* As a part of the Monthly pricing phase 4, this PR enables the monthly plans to the My Plans page. For more details, see pcbrnV-1gA-p2

#### Testing instructions



First of all, the monthly pricing is available only for free users and monthly plan users. You should make sure the following changes don't affect on any pages as long as you are a non-monthly plan user.

* If your site is either on the free plan or a monthly plan, My Plans page should display the Interval type selector (red box) and all four paid plans should show up in a row.
  <img width="1024" alt="Plans_‹_PremiumSite_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/108209282-76985200-716d-11eb-8e85-e523ea3cf6e3.png">
* With a non-monthly paid plan, the Customer type selector should appear instead of the Interval type selector.
  <img width="800" alt="Plans_‹_Taggon_on_wordpress_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/107378112-00716b00-6b2f-11eb-9dc1-eedf47679090.png">
* Make sure the both site selectors work as expected.
* Check the `plans` step of the new site creation flow. It should not change.
  <img width="800" alt="Create_a_site_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/107378619-7ece0d00-6b2f-11eb-8d9e-fdbb6bb58c81.png">
